cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-logging.adoc

```diff
@@ -0,0 +1,219 @@
+
+[NOTE]
+====
+ROSA is not preconfigured with a logging solution.
+====
+
+The cluster logging components are based on Fluentd, Elasticsearch (ES), and Kibana. Fluentd is deployed to each node in the cluster. It collects application logs and either writes them to ES or forwards them to CloudWatch. Kibana is a centralized, web-based user interface (UI) where users and administrators can create visualizations and dashboards with the aggregated data.
+
+.Prerequisites
+
+* Set up a logging solution before viewing your logs. See the xref:../../observability/logging/cluster-logging-deploying.adoc#cluster-logging-deploying[Installing logging] documentation for more information.
+
+.Additional resources
+* xref:../../observability/logging/cluster-logging.adoc#cluster-logging[Cluster logging]
+
+== Installing the logging add-on service
+
+Install the logging add-on service to forward the logs to AWS CloudWatch.
+
+//These steps were adopted from the Managed OpenShift Black Belts xref:../../cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts.adoc#cloud-experts-rosa-cloudwatch-sts[tutorial].
+
+. Run the following script to configure your ROSA cluster to forward logs to CloudWatch:
++
+[NOTE]
+====
+Configuring ROSA to send logs to CloudWatch goes beyond the scope of this tutorial. Integrating with AWS and enabling CloudWatch logging are important aspects of ROSA, so a script has been included to simplify the configuration process. The script will automatically set up AWS CloudWatch. You can examine the script to understand the steps involved.
+====
++
+[source,terminal]
+----
+$ curl https://raw.githubusercontent.com/openshift-cs/rosaworkshop/master/rosa-workshop/ostoy/resources/configure-cloudwatch.sh | bash
+----
++
+.Example output
++
+[source,terminal]
+----
+deployment "cluster-logging-operator" successfully rolled out
+secret/cloudwatch-credentials created
+clusterlogforwarder.logging.openshift.io/instance created
+clusterlogging.logging.openshift.io/instance created
+----
+. After a few minutes, you should begin to see log groups inside of AWS CloudWatch. Run the following command to see the log groups:
++
+[source,terminal]
+----
+$ aws logs describe-log-groups --log-group-name-prefix rosa-${GUID}
+----
++
+.Example output
++
+[source,terminal]
+----
+{
+	"logGroups": [
+		{
+			"logGroupName": "rosa-fxxj9.audit",
+			"creationTime": 1682098364311,
+			"metricFilterCount": 0,
+			"arn": "arn:aws:logs:us-east-2:511846242393:log-group:rosa-fxxj9.audit:*",
+			"storedBytes": 0
+		},
+		{
+			"logGroupName": "rosa-fxxj9.infrastructure",
+			"creationTime": 1682098364399,
+			"metricFilterCount": 0,
+			"arn": "arn:aws:logs:us-east-2:511846242393:log-group:rosa-fxxj9.infrastructure:*",
+			"storedBytes": 0
+		}
+	]
+}
+----
+
+== Outputting the data to the streams and logs
+. Output a message to `stdout`.
+.. In the OSToy console, click *Home* and then click the message box for *Log Message (stdout)*.
```

Comment:
I don't know what OSToy console is?  Can you ask the owner review this?

Reply:
This PR is part of a longer multi-page procedure. The users have set up the OSToy console in a previous step and will know what it is.